CHANGES

```diff
@@ -1,3 +1,34 @@
+# 4.0.0
+
+* Implemented an MCVirt daemon that accepts commands through RPC.
+* Implemented SSL support instead of using SSH.
```

Comment:
This is not strictly true. SSH was for communications more than security... although i did that as well..
I would say switch to using 'RPC-over-SSL' from 'a custom command script using SSH'